scripts/config/templates/nginx.conf.erb

```diff
@@ -312,7 +312,7 @@ http {
   }
 
   # redirect all commercial, facility, location, and storage pages to ssr-frontend
```

Comment:
Out of curiosity, is it worth changing the comment to reflect these changes?

Reply:
Oh yeah good point! I'll go change that